db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,40 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
```

Comment:
We should keep the same trend going here:  `container_deployment_authentications`